samples/src/knot.cpp

```diff
@@ -44,7 +44,7 @@ Manifold TorusKnot(int p, int q, float majorRadius, float minorRadius,
   p /= kLoops;
   q /= kLoops;
   int n = circularSegments > 2 ? circularSegments
-                               : Manifold::GetCircularSegments(threadRadius);
+                               : Quality::GetCircularSegments(threadRadius);
```

Comment:
This is a good opportunity to switch to using your new `Circle()` method.

Reply:
Oh I thought I got all of the `Manifold::GetCircularSegments` uses.. weird, must have messed up my regex. Do you find any others or is that the last hanger on? I got rid of the Manifold ones so I figured the compiler would catch me for missing any.

Reply:
Ah good point. Cylinder as well.

Reply:
I've converted all of the manual circle creations that I found :+1: